2ls.inc

```diff
@@ -10,14 +10,26 @@ run()
   # add property-specific options
   if [[ "$PROP" == "termination" ]]; then
     PROPERTY="$PROPERTY --termination --competition-mode"
+    PROPERTY2="$PROPERTY --nontermination --competition-mode"
+    
+    # run the tool
```

Comment:
extend the comments to explain the logic behind this (which result counts in the end)